docs/rst/tutorials/core/wifi/image_compression/image_compression.rst

```diff
@@ -0,0 +1,235 @@
+.. include:: ../../../../exports/alias.include
+
+.. _tutorials_image_compression:
+
+How to compress images for video streaming in ROS 2
+===================================================
+
+.. contents::
+    :depth: 2
+    :local:
+    :backlinks: none
+
```

Comment:
Los otros tutoriales de "Vulcanexus WiFi & Large Data Tutorials" tienen una primera subsection de Have you ever have this problem? y con una "Quick Solution Overview". Este tutorial es el unico que no tiene